crates/config/src/lib.rs

```diff
@@ -302,9 +305,10 @@ pub struct Config {
     /// Multiple etherscan api configs and their aliases
     #[serde(default, skip_serializing_if = "EtherscanConfigs::is_empty")]
     pub etherscan: EtherscanConfigs,
-    /// list of solidity error codes to always silence in the compiler output
+    /// List of solidity error codes to always silence in the compiler output.
     pub ignored_error_codes: Vec<SolidityErrorCode>,
-    /// list of file paths to ignore
+    /// List of file paths to ignore.
+    // todo(onbjerg): how is this different from `skip`?
     #[serde(rename = "ignored_warnings_from")]
     pub ignored_file_paths: Vec<PathBuf>,
```

Comment:
this is for ignoring warnings https://github.com/foundry-rs/compilers/blob/799f83ed4a1b67e39428106f51128d1c8f891f8c/crates/compilers/src/compile/output/mod.rs#L831